src/mlpack/core/data/CMakeLists.txt

```diff
@@ -24,6 +24,8 @@ set(SOURCES
   split_data.hpp
   imputer.hpp
   binarize.hpp
+  feature_selection.hpp
+  feature_selection_imp.hpp
```

Comment:
For consistency we should go with `impl.hpp`.

Reply:
renamed everywhere